### PR TITLE
Suite Cleanup 2: Fixing a Ton of Internal Typos: Phase 1

### PR DESCRIPTION
**Pull request by Windchild292**
This fixes a ton of typos and some formatting issues

### CODE REVIEW

File: MekHQ/data/mechfiles/mechs/3039u/Clint CLNT-2-4T.mtf [hunk 1]
```diff
@@ -153,7 +153,7 @@ Overview: Not every BattleMech developed for the Star League was a  technologica
...
+Deployment: The Clint was found in every army in the Inner Sphere during the Succession Wars. No factories produced the design, but notable concentrations were found in the Federated Commonwealth and Capellan Confederation. Certain other states, such as the Free Worlds League, often used their Clints for little more than spare parts. A modification made by the two major users of the design, the -2-4T sacrificed manoeuvrability and short-ranged firepower in order to increase the number of autocannons it carried. Commonly used against VTOLs, tanks, and other conventional forces, it posed little threat against 'Mechs able to catch up to it. Just like the base Clint, no -2-4T's remained in the armies of House Davion of Liao (or any other Houses) by the end of the Jihad.
```
Comment (rjhancock) [inline MekHQ/data/mechfiles/mechs/3039u/Clint CLNT-2-4T.mtf:156]: Doing a quick look shows this changes the definition of sentence and is not an actual spelling error.

File: MekHQ/data/universe/news.xml [hunk 2]
```diff
@@ -22,1912 +22,1912 @@ location - an optional tag for the location of the news report
...
-		<service>CBN</service>
-		<location>Ashio</location>
-	</newsItem>
-	<newsItem>
-		<headline>Parliament seeks restitution from First Lord and Captain-General</headline><!-- Source: House Marik (The Free Worlds League) -->
-		<date>2989-10</date>
-		<desc><![CDATA[<p>Filed 12 September 2989</p><p>Parliament is setting a new precident by demanding restitution from the First Lord and Captain-General and the military as a whole to replenish funds used during the recent wars. Special Resolution 512 calls for funds, materials, and equipment to be disbursed back into the FWL economy to help bolster those industries most effected by the fighting. No official word from the office of the First Lord. Economists are following this news intently as these reparations could significantly bolster struggling economies across the FWL.</p>]]></desc>
```
Comment (rjhancock) [inline MekHQ/data/universe/news.xml:290]: precedent I believe vs establishing a new ruling member

File: MekHQ/data/mechfiles/mechs/3039u/Clint CLNT-2-4T.mtf (hunk 1 above)
Comment (rjhancock) [reply MekHQ/data/mechfiles/mechs/3039u/Clint CLNT-2-4T.mtf:156]: Disregard this one. Both are valid.

Comment (Windchild292) [reply MekHQ/data/mechfiles/mechs/3039u/Clint CLNT-2-4T.mtf:156]: `manueverability` is a typo, `manoeuvrability` is the correct spelling.

Comment (rjhancock) [reply MekHQ/data/mechfiles/mechs/3039u/Clint CLNT-2-4T.mtf:156]: It is a typo but one of a valid variation of the word: https://www.merriam-webster.com/dictionary/maneuverability

**Review by rjhancock — APPROVED** [review]

File: MekHQ/data/universe/news.xml (hunk 2 above)
Comment (Windchild292) [reply MekHQ/data/universe/news.xml:290]: Both are mistakes... it should be precedent as you stated.

**Pull request merged**